.tekton/tasks/install-npm-dependencies.yaml

```diff
@@ -34,6 +38,11 @@ spec:
         ARTIFACTS_PATH="$(workspaces.output.path)"
         cd $ARTIFACTS_PATH
 
+        if [ -n "$(params.prerelease)" ]; then
+          chmod +x ./.tekton/install-node.sh
+          ./.tekton/install-node.sh "$(params.prerelease-node-version)"
+        fi
+
         if [ -n "$(params.npm-version)" ]; then
```

Comment:
The current nighlty version of node is not compatible to download all the npm modules in the package.
Expecting RC version to resolve this issue .

Reply:
Please share more context.

> nighlty version of node is not compatible to download

Reply:
I assume your comment is related to
https://github.com/instana/nodejs/pull/1358/files#diff-665bda1af67cd19ff7a8dc1904d798a45d36f190ebf014b3850ffc6adc9133a2R41-R46

Reply:
Yes correct. Currently we are using node modules installed using the containers node version.
Once RC version is released, I can check and confirm the node modules installation later

Reply:
> nighlty version of node is not compatible to download

Why?

Reply:
There is a warning 
`npm warn cli npm v10.9.0 does not support Node.js v23.0.0-pre. This version of npm supports the following node versions: `^18.17.0 || >=20.5.0`. You can find the latest version at https://nodejs.org/.`

and also, node-gyp fails with error
```
npm error gyp info using node@23.0.0-pre | darwin | arm64
npm error gyp ERR! stack Error: "pre" versions of node cannot be installed, use the --nodedir flag instead
```

This is coming from `info run @instana/autoprofile@3.20.0 install packages/autoprofile node-gyp-build`

ps: other packages that used node-gyp might also fail because of the node version

Reply:
And what only happens if you are using "nightly"?

Reply:
@abhilash-sivan ?

Reply:
Yes, it happens because of the incompatibility of the node-version and npm-version.

Reply:
Did you try `npm install --force`?

Reply:
Added
>    "install-deps-prerelease": "lerna exec --ignore packages/autoprofile -- rm -rf node_modules && npm install --loglevel verbose --force",

We can ignore autoprofiler pkg for now.
Installs now.